x-pack/legacy/plugins/lens/public/indexpattern_plugin/operation_definitions/terms.tsx

```diff
@@ -70,14 +78,31 @@ export const termsOperation: OperationDefinition<TermsIndexPatternColumn> = {
       field: column.sourceField,
       orderBy:
         column.params.orderBy.type === 'alphabetical' ? '_key' : column.params.orderBy.columnId,
-      order: 'desc',
+      order: column.params.orderDirection,
       size: column.params.size,
       otherBucket: false,
       otherBucketLabel: 'Other',
       missingBucket: false,
       missingBucketLabel: 'Missing',
     },
   }),
+  onOtherColumnChanged: (currentColumn, columns) => {
+    if (currentColumn.params.orderBy.type === 'column') {
+      // check whether the column is still there and still a metric
+      const columnSortedBy = columns[currentColumn.params.orderBy.columnId];
+      if (!columnSortedBy || columnSortedBy.isBucketed) {
+        return {
+          ...currentColumn,
+          params: {
+            ...currentColumn.params,
+            orderBy: { type: 'alphabetical' },
+            orderDirection: 'asc',
```

Comment:
I would expect the default here to be count of documents, even if that isn't explicitly rendered to the user. Is that possible?